projects/netgrif-components-core/src/lib/authentication/services/methods/basic-authentication/basic-authentication.service.ts

```diff
@@ -5,6 +5,7 @@ import {AuthenticationMethodService} from '../../authentication-method.service';
 import {Credentials} from '../../../models/credentials';
 import {ConfigurationService} from '../../../../configuration/configuration.service';
 import {UserResource} from '../../../../resources/interface/user-resource';
+import * as Buffer from 'buffer';
```

Comment:
where is `Buffer` imported from? I assume it is not a native JS method, since it needs importing, but no dependency was added to the project.

Reply:
This is an official Node.js object for working with strings and byte operations.

Reply:
does it work in the browser if it is a Node.js class?

Reply:
Yes, it does. I have tested it in several ways: on standard login view and on console of Google Chrome and Safari, too.